app/policies/application_policy.rb

```diff
@@ -0,0 +1,14 @@
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  protected
+
+  def manager?
+    user.role == "manager"
```

Comment:
btw, I think as you are using enum for roles, `user.manager?` would work as well...

Reply:
Yep, it does :). Changed it.